src/app/panels/tab-proxy.js

```diff
@@ -20,6 +20,7 @@ export class TabProxy {
 
     fileManager.events.on('fileClosed', (name) => {
       this.removeTab(name)
+      this.switchPreviousTab()
```

Comment:
if you want to switch to home when no tab are displayed, that's not the right solution.